soccer/src/soccer/joystick/sdl_joystick_node.cpp

```diff
@@ -1,16 +1,14 @@
 #include "sdl_joystick_node.hpp"
 
-#include <iostream>
-
 #include <ament_index_cpp/get_package_share_directory.hpp>
+#include <spdlog/spdlog.h>
 
 namespace joystick {
 SDLJoystickNode::SDLJoystickNode(Context* context) : context_{context} {
     // initialize using the SDL joystick
     if (SDL_Init(SDL_INIT_GAMECONTROLLER | SDL_INIT_EVENTS) != 0) {
-        std::cerr << "ERROR: SDL could not initialize game controller system! SDL "
-                     "Error: "
-                  << SDL_GetError() << std::endl;
+        SPDLOG_ERROR("SDL could not initialize game controller system! SDL Error: {}",
+                     SDL_GetError());
         throw std::runtime_error("");
```

Comment:
Debug throw? Also, lmao empty exception string.

Reply:
Changed to `fatal_throw`.